src/server/HelixImportServer.js

```diff
@@ -100,6 +109,7 @@ export class HelixImportServer extends BaseServer {
     delete headers.connection;
     delete headers.host;
     delete headers.referer;
+    await this._updateHeaders(headers);
```

Comment:
you read the headers for each request? why don't you cache it on startup?

Reply:
Good point. I addressed it at https://github.com/adobe/helix-cli/pull/2368/commits/a302dc738a6a2a448b84521404f5bc32d2e32260